modules/store/testing/src/mock_store.ts

```diff
@@ -36,6 +36,7 @@ export class MockStore<T> extends Store<T> {
   >();
 
   public scannedActions$: Observable<Action>;
+  private lastState: T | undefined;
```

Comment:
```suggestion
  private lastState: T;
```

I think `lastState` will always be provided (or is an empty object)?
Because of `config.initialState || {}`

Reply:
Yep. I initially didn't have the `|| {}` there.